File: 0997-find-the-town-judge/0997-find-the-town-judge.cpp
class Solution {
public:
    int findJudge(int n, vector<vector<int>>& trust) {
        vector<int> count(n + 1, 0);
        for(auto arr : trust){
            count[arr[0]]--;
            count[arr[1]]++;
        }
        for(int i = 1; i < count.size(); i++){
            if(count[i] == n - 1) return i;
        }
        return -1;
    }
};

File: 0997-find-the-town-judge/0997-find-the-town-judge.java
class Solution {
    public int findJudge(int n, int[][] trust) {
        int[] count = new int[n + 1];
        for(int i = 0; i < trust.length; i++){
            count[trust[i][0]]--;
            count[trust[i][1]]++;
        }
        for(int i = 1; i < count.length; i++){            
            if(count[i] == n - 1) {
                return i;
            }
        }
        return -1;
    }
}